src/Umbraco.Commerce.Checkout/Extensions/CompositionExtensions.cs

```diff
@@ -36,8 +35,6 @@ public static IBuilder AddUmbracoCommerceEventHandlers(this IBuilder builder)
 
             // Toggle order editor shipping address enabled flag based on
             // whether there umbraco commerce checkout is configured to collect a shipping address
```

Comment:
I think we can convert the above handler registrations to be

````
builder.WithUmbracoCommerceBuilder().WithNotificationEvent<>().RegisterHandler<>();
````

This should allow us to remove the pragma statements